CONTRIBUTING.rst

```diff
@@ -10,10 +10,7 @@ Git workflow
 ------------
 
 The WarpX project uses `git <https://git-scm.com>`_ for version control.
-If you are new to git, you can follow one of these tutorials:
-
-- `Learn git with bitbucket <https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud>`_
-- `git - the simple guide <http://rogerdudler.github.io/git-guide/>`_
+If you are new to git, you can follow `this tutorial <https://swcarpentry.github.io/git-novice/>`_.
```

Comment:
Anonymous link because link text is quite generic:
```suggestion
If you are new to git, you can follow `this tutorial <https://swcarpentry.github.io/git-novice/>`__.
```